nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/io/ExtendedInfoSerializer.java

```diff
@@ -57,22 +57,30 @@ public void serialize(ExtendedInfo info, JsonGenerator jg,
             Serializable value = ((ExtendedInfoImpl.BlobInfo) info).getBlobValue();
             jg.writeObject(Base64.encodeBase64(SerializationUtils.serialize(value)));
         } else if (info instanceof ExtendedInfoImpl.StringInfo) {
-            String stringValue = ((ExtendedInfoImpl.StringInfo) info).getStringValue().trim();
-            if ((stringValue.startsWith("{") && stringValue.endsWith("}"))
-                    || (stringValue.startsWith("[") && stringValue.endsWith("]"))) {
-                try {
-                    mapper.readTree(stringValue);
-                    jg.writeRawValue(stringValue);
-                } catch (IOException e) {
-                    // If the value represents an invalid JSON, send a null value to ES to prevent potential
-                    // mapping exceptions
-                    jg.writeObject(null);
-                }
+            writeString(jg, mapper, ((ExtendedInfoImpl.StringInfo) info).getStringValue().trim());
+        } else {
```

Comment:
Please add a comment explaining what `info` is in that case, if it's not a date/blob/string info.